platforms/extensibility/plugin-development/src/integTest/groovy/org/gradle/plugin/devel/plugins/JavaGradlePluginPluginPublishingIntegrationTest.groovy

```diff
@@ -260,6 +261,90 @@ class JavaGradlePluginPluginPublishingIntegrationTest extends AbstractIntegratio
         ivyRepo.module('com.example.foo', 'com.example.foo' + PLUGIN_MARKER_SUFFIX, 'unspecified').assertPublished()
     }
 
+    @Issue("https://github.com/gradle/gradle/issues/12259")
+    def "when publishing to maven then name and description from plugin block are used by convention when writing pom"() {
```

Comment:
Looks like this test and `Publishes one Maven marker for every plugin` tests the same functionality